test/account_test.rb

```diff
@@ -90,17 +90,18 @@ def app
     end
 
     it "shows a helpful page for a token that wasn't found" do
+      any_instance_of(Identity::Cookie) do |cookie|
+        stub(cookie).access_token { "abc123" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.